### DISCUSSION TITLE
Step by Step for newbies to postcss

### DISCUSSION THREAD

**Discussion by alexishughes** [tailwindlabs/tailwindcss#3270] [Help] [answered]
Hi. My css knowledge is rusty. Of course, I get that I could just link the tailwindcss file to my page and it would work.

What I want to do is use postcss to create a minimal tailwind css style for my project (C# dotnet core 5 EF MVC) - I can make all the layout files etc. easy.

Question is: Literally, what do I type in Powershell to install (Ok. documentation tells me)? How do I build/rebuild my css files when I add components, what javascript do I need to include with the layout page to ensure ?postcss ?anything else is running.

I'd love you to create a blank HTML project with all the files in place and post the npm commands in the readme. Then I can hack this into VS for MVC dotnet core.

Many thanks in advance, and apologies if this is a trivial question.

**Comment by alexishughes** [comment, marked as answer]
https://www.youtube.com/watch?list=PL7CcGwsqRpSM3w9BT_21tUU8JN2SnyckR&v=21HuwjmuS7A&index=1
This looks concise. Ty.

**Reply by simonswiss** [reply]
Yes - I was going to point you to exactly that first video!

Some things have changed in Tailwind since this course was recorded (I am actually in the process of creating a refresh of this video series) - but the info on using `postcss-cli` to compile Tailwind through PostCSS is still totally relevant! 👍
